ci/run-linter.sh

```diff
@@ -2,12 +2,18 @@
 set -e
 set -eo pipefail
 
-echo "Code Styling with (black and flake8)"
+echo "Code Styling with (black, flake8, isort)"
 
 source activate xskillscore-dev
 
 echo "[flake8]"
-flake8 xskillscore --max-line-length=88 --exclude=__init__.py --ignore=W605,W503,C901,E711
+flake8 xskillscore
```

Comment:
Riley was in favor of These 88 Max. I don’t care. Can leave as is.

Reply:
I added E711. This is needed for that one ==None check. You will see probably. If not then don’t include.

Reply:
Am I think it thinking the flags are superfluous as they are picked up in https://github.com/raybellwaves/xskillscore/blob/add-pre-commit/.pre-commit-config.yaml#L18 or do I still need them

Reply:
ah I guess these are post-commit. Will add back in.